pages/How-to/guide/create-tags.md

```diff
@@ -1,3 +1,32 @@
-![WORK IN PROGRESS](https://user-images.githubusercontent.com/51878265/188319718-b341b07b-ef84-4aa3-a8aa-722eb52aff7e.gif)
+# What are Github Tags ?
+Tags are associated with commits, so you can use a tag to mark an individual point in your repository's history, including a version number for a release.
 
-<h4 align="center">If you want to contribute to this page. Please check out the <a href="https://github.com/Pradumnasaraf/open-source-with-pradumna/issues">issues</a></h4>
\ No newline at end of file
+# How to create Tags ?
+
+### Step 1
+
+You should have a repository, if you don't go on <u><a href = "https://oswp.study/#/pages/How-to/guide/repo-private-public">this</a></u> page to know how to create a repository.
+
+### Step 2
+
+Navigate to the repository and click on tags <br>
+<img width="341" alt="image" src="https://user-images.githubusercontent.com/100528412/193890608-85e902d5-5f50-49a6-8ad5-4f3370530a25.png">
+
+### Step 3 
+
+Click on `Create a new release` <br> <img width="914" alt="image" src="https://user-images.githubusercontent.com/100528412/193890768-fb29e090-5d07-48bb-92b9-0f860f75d490.png">
+
+### Step 4
+
+Click on `Choose a tag` to choose a tag and `target:` to choose the branch you are targetting.
```

Comment:
I think here we need a screenshot to show how it looks.

Reply:
Okay will make those changes in a min !